dts/riscv/starfive/starfive_jh7100_beagle_v.dtsi

```diff
@@ -0,0 +1,190 @@
+#include "starfive_jh7100_clk.dtsi"
```

Comment:
It's possible that this might be better in the board directory rather than `dts/`

Reply:
I am not sure about this one tbh. I think this dts belongs to soc rather than the board. I can rename the dts file if that makes sense. Something like starfive_jh7100.dtsi. This one is taken from [here](https://github.com/starfive-tech/u-boot/tree/Fedora_VIC_7100_2021.04/arch/riscv/dts) and so the name follows.